### ISSUE TITLE
`supabase status` outputs only the DB URL and shows most services as stopped

### ISSUE THREAD

**Issue by hammerlscs** [supabase/cli#2419]
**Describe the bug**
The command `supabase status` outputs only the DB URL and shows most services as stopped.
The problem is probably related to how the Supabase CLI checks whether a service is healthy and possibly related to these changes: https://github.com/supabase/cli/commit/72cf66912043169ab3cc09c85d914b3570c8a8a9

Version 1.176.4 (latest):
```
npx supabase status

Using workdir [removed on purpose]
Stopped services: [supabase_kong_dashboard supabase_auth_dashboard supabase_inbucket_dashboard supabase_realtime_dashboard supabase_rest_dashboard supabase_storage_dashboard supabase_imgproxy_dashboard supabase_pg_meta_dashboard supabase_studio_dashboard supabase_edge_runtime_dashboard supabase_analytics_dashboard supabase_vector_dashboard supabase_pooler_dashboard]
supabase local development setup is running.

    DB URL: postgresql://postgres:postgres@127.0.0.1:5492/postgre
```

Version 1.172.2:
```
npx supabase status

Stopped services: [supabase_analytics_dashboard]
supabase local development setup is running.

    API URL: http://127.0.0.1:5491
    GraphQL URL: http://127.0.0.1:5491/graphql/v1
...
    S3 Region: local
A new version of Supabase CLI is available: v1.176.4 (currently installed v1.172.2)
We recommend updating regularly for new features and bug fixes: https://supabase.com/docs/guides/cli/getting-started#updating-the-supabase-cli
```
(While our Supabase project uses different ports, the problem persisted when testing it with the default ports.)

**The command `supabase start` still outputs the expected information:**

Version 1.176.4 (latest):
```
npx supabase start

Using workdir [removed on purpose]
Started supabase local development setup.

    API URL: http://127.0.0.1:5491
    GraphQL URL: http://127.0.0.1:5491/graphql/v1
...
    S3 Region: local
```

Version 1.172.2:
```
npx supabase start

Started supabase local development setup.

    API URL: http://127.0.0.1:5491
    GraphQL URL: http://127.0.0.1:5491/graphql/v1
...
    S3 Region: local
A new version of Supabase CLI is available: v1.176.4 (currently installed v1.172.2)
We recommend updating regularly for new features and bug fixes: https://supabase.com/docs/guides/cli/getting-started#updating-the-supabase-cli
```

**To Reproduce**
1. `supabase init`
2. `cd supabase`
3. `supabase start`
4. `supabase status`

**Expected behavior**
The output of `supabase status` should include additional information such as API URL, Studio URL, ...

**System information**
 - Version of OS: macOS 14.5 (also Arch Linux 6.8.9, Ubuntu 22.04 and Bitbucket Pipeline)
 - Version of CLI: 1.176.4
 - Version of Docker: 4.31.0

**Comment by jonaias-sn** [comment]
Same problem here. I use it to generate `.env` for Github Actions testing

**Comment by sweatybridge** [comment]
Thanks for reporting this issue. I've released a fix to beta channel: v1.176.7

**Comment by hammerlscs** [comment]
Thank you for the quick fix, @sweatybridge!
